game/scripts/vscripts/components/boss/ngp.lua

```diff
@@ -5,8 +5,13 @@ if NGP == nil then
     NGP = class({})
 end
 
-NGP.itemIndex = 0
+NGP.itemIndex = 2
 NGP.activeItems = {}
+NGP.activeTimers = {}
+local totalgoodplayers = 0
+local totalbadplayers = 0
+local ngpItemsGood = {}
+local ngpItemsBad = {}
```

Comment:
These items are local things, they can be re-retrieved in every other class using standart methods, they dont need to be global